app/models/user.rb

```diff
@@ -1,4 +1,5 @@
 class User < ApplicationRecord
+  has_many :groups, class_name: 'group', foreign_key: 'owner_id'
```

Comment:
I think we have to find a better name here too to represent the groups that this user is owner, because we are going to have a relation with the same name in the future.

We could have something like:

```ruby
user.own_groups # the groups that the user is owner

user.groups # groups that the user is part of
```

What do you think?

Reply:
How about...
`user.owned_groups`
It translates to 'grupos propios'